packages/dd-trace/src/config.js

```diff
@@ -136,6 +136,11 @@ class Config {
       process.env.DD_DBM_PROPAGATION_MODE,
       'disabled'
     )
+    const DD_DATA_STREAMS_ENABLED = coalesce(
```

Comment:
This should be made into a boolean instead of a string, that way we don't need the `!== 'disabled'` in multiple places.